tap_google_sheets/client.py

```diff
@@ -145,6 +146,12 @@ def __init__(self,
         self.__expires = None
         self.__session = requests.Session()
         self.base_url = None
+        self.supports_all_drives = False
+
+        # if supports_all_drives is given as true then use supports_all_drives
+        if supports_all_drives in ('true', 'True', True):
```

Comment:
You can directly check the if condition.
```suggestion
        if supports_all_drives:
```